CONTRIBUTING.md

```diff
@@ -151,32 +151,42 @@ To submit a [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-req
 ### Using Conventional Commits
 
 Please help the maintainers by leveraging the following [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/)
-standards in your pull request title and commit messages.
+standards in your commit messages. You don't need to apply this format to the pull request title–just to commits.
+
+As a reminder, here's the format for a single-line commit, but you are welcome to add the optional body and footer messages.
+
+```
+<type>(optional scope): <description>
+```
+
+When choosing a type, you can pick from any of the standard types (feat, fix, style, test, or chore) or you can add your own. If you are making document changes, as opposed to code changes, you can skip the `documentation` type and apply one of the other types, depending on what you're changing in the documentation.
```

Comment:
The second sentence here is a bit confusing, maybe we could say something along these lines:
```suggestion
When choosing a type, you can pick from any of the standard types (feat, fix, style, test, or chore) or you can add your own. 
**Note**: There is a `documentation` type, but please refrain from using it for general documentation changes since it is intended for changes to the repository's documentation (such as `README.md` and `CONTRIBUTING.md`) 
```